CONTRIBUTING.md

```diff
@@ -72,6 +73,7 @@ requests_).
  - Ronnie Dutta
  - John Haiducek
  - (Andrew Huang)
```

Comment:
CONTRIBUTING.md vs commit authors:
```diff
0a1
> Alex Reinecke
29,30d29
< Patrick Alexander Reinecke
< Patrick Reinecke
41d39
< hilary.oliver
```

Might we need a JSON file or something that maps aliases of people to one name?

Reply:
That's what the `.mailmap` file does.

Reply:
Already done on this PR ☕